doc/source/building/working_with_images/setup_pxe_bootserver.rst

```diff
@@ -40,9 +41,10 @@ serve as an example. Depending on your network structure, the IP addresses,
 ranges and domain settings need to be adapted to allow the DHCP server to
 work within your network. If you already have a DHCP server running in your
 network, make sure that the filename and next-server are correctly set
```

Comment:
what are these `filename` and `next-server`? Are these options, parameters etc? In that case, I'd recommend to add the term "parameter" or "option" after these and enclose the terms in backticks.